### DISCUSSION TITLE
К кому можно обратиться с просьбой правки zenengine?

### DISCUSSION THREAD

**Discussion by LDS1** [dosinabox/g2nr_unofficial_update#420] [Q&A] [answered]
сабж https://github.com/SaiyansKing/GD3D11/issues/3#issuecomment-992049262

**Comment by dosinabox** [comment, marked as answer]
Gratt или Slavemaster на wop. Или отредактировать сами частицы, если они содержат инфу об освещении.

**Comment by dosinabox** [comment]
Лучше сразу Slavemaster, он занимался частицами и эффектами в своих плагинах.
